tbk/onepay/__init__.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
```

Comment:
You are right. First, we're not supporting py2x, only >=py34, `absolute_import` or `unicode_literals` are not necessary because it covers behaviours that are the default in py3.

So, I'm agree, and the commit 9722c9c fix this.